cmd/gateway-sia.go

```diff
@@ -20,6 +20,9 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/minio/cli"
+	"github.com/minio/minio-go/pkg/set"
+	"github.com/minio/minio/pkg/hash"
```

Comment:
Keep the imports in their previous location here. @dvstate you should run `goimports` on gateway-sia.go for proper styling.